index.html

```diff
@@ -61,29 +61,25 @@ <h1>MusicTripper - zaplanuj swoją podróż muzyczną</h1>
         <div> 
           <img src="./img/jiraya.jpg" alt="OUR PICTURE" class="teammate_picture">
         </div>
-        <div>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quas culpa aliquid facilis adipisci! Dolorem, ullam accusantium, sapiente architecto, ducimus quo impedit dicta quas non in odit perspiciatis recusandae explicabo eveniet?</div>
-        <div>PROFILE LINK</div>
+        <div>Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsa iusto ut, eos eius repellendus amet, ea nesciunt nostrum dolorum beatae odit, non temporibus in assumenda illo quis earum aliquid numquam. <a href="">PROFILE LINK</a></div>
       </div>
       <div class="teammate">
         <div>
           <img src="./img/jiraya.jpg" alt="OUR PICTURE" class="teammate_picture">
         </div>
-        <div>Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsa iusto ut, eos eius repellendus amet, ea nesciunt nostrum dolorum beatae odit, non temporibus in assumenda illo quis earum aliquid numquam.</div>
-        <div>PROFILE LINK</div>
+        <div>Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsa iusto ut, eos eius repellendus amet, ea nesciunt nostrum dolorum beatae odit, non temporibus in assumenda illo quis earum aliquid numquam. <a href="">PROFILE LINK</a></div>
       </div>
       <div class="teammate">
         <div>
           <img src="./img/jiraya.jpg" alt="OUR PICTURE" class="teammate_picture">
         </div>
-        <div>Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsa iusto ut, eos eius repellendus amet, ea nesciunt nostrum dolorum beatae odit, non temporibus in assumenda illo quis earum aliquid numquam.</div>
-        <div>PROFILE LINK</div>
+        <div>Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsa iusto ut, eos eius repellendus amet, ea nesciunt nostrum dolorum beatae odit, non temporibus in assumenda illo quis earum aliquid numquam. <a href="">PROFILE LINK</a></div>
       </div>
       <div class="teammate">
           <div> 
             <img src="./img/jiraya.jpg" alt="OUR PICTURE" class="teammate_picture">
           </div>
-          <div>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quas culpa aliquid facilis adipisci! Dolorem, ullam accusantium, sapiente architecto, ducimus quo impedit dicta quas non in odit perspiciatis recusandae explicabo eveniet?</div>
-          <div>PROFILE LINK</div>
+          <div>Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsa iusto ut, eos eius repellendus amet, ea nesciunt nostrum dolorum beatae odit, non temporibus in assumenda illo quis earum aliquid numquam. <a href="">PROFILE LINK</a></div>
         </div>
     </section>
 
```

Comment:
Please use code formatter in VSC (right click on code and chose "Format Document")